src/components/AdminPanelEntry/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Proptypes from "prop-types";
+
+export default function AdminPanelEntry({ title, placeholder }) {
+  const [name, setName] = useState("");
+
+  const handleKeyDown = (ev) => {
+    const enter = 13;
+    if (ev.keyCode === enter) {
+      const { value } = ev.target;
+      // socket.emit("add_channel", {
+      //  channel: name
+      // });
+
+      setName("");
+    }
+  };
+
+  const onChange = (ev) => {
+    setName(ev.target.value);
+  };
```

Comment:
```suggestion
  const handleOnChange = (ev) => {
    setName(ev.target.value);
  };
```

함수명 handleOnChange 혹은 handleChange 제안드립니다!